src/d3a/models/area/__init__.py

```diff
@@ -190,23 +191,29 @@ def _cycle_markets(self, _trigger_event=True, _market_cycle=False, deactivate=Fa
                 and _trigger_event and ConstSettings.BalancingSettings.ENABLE_BALANCING_MARKET:
             self.dispatcher.broadcast_balancing_market_cycle()
 
-    def tick(self, is_root_area=False):
+    def tick(self):
```

Comment:
Was `is_root_area` ever used?

Reply:
yes, some months ago. But when not needed any more, it wasnt deleted here.